
CommandNotFoundError: Your shell has not been properly configured to use 'conda deactivate'.
To initialize your shell, run

    $ conda init <SHELL_NAME>

Currently supported shells are:
  - bash
  - fish
  - tcsh
  - xonsh
  - zsh
  - powershell

See 'conda init --help' for more information and options.

IMPORTANT: You may need to close and restart your shell after running 'conda init'.


Traceback (most recent call last):
  File "/gpfs/loomis/project/lederman/dge5/conda_envs/cryodrgn_rotations/bin/cryodrgn", line 33, in <module>
    sys.exit(load_entry_point('cryodrgn==1.0.0b0', 'console_scripts', 'cryodrgn')())
  File "/gpfs/loomis/project/lederman/dge5/conda_envs/cryodrgn_rotations/lib/python3.7/site-packages/cryodrgn-1.0.0b0-py3.7.egg/cryodrgn/__main__.py", line 58, in main
    args.func(args)
  File "/gpfs/loomis/project/lederman/dge5/conda_envs/cryodrgn_rotations/lib/python3.7/site-packages/cryodrgn-1.0.0b0-py3.7.egg/cryodrgn/commands/train_vae.py", line 513, in main
    print(ztracker.get_zval(torch.Tensor([0], dtype=torch.int32)))
TypeError: new() received an invalid combination of arguments - got (list, dtype=torch.dtype), but expected one of:
 * (*, torch.device device)
      didn't match because some of the keywords were incorrect: dtype
 * (torch.Storage storage)
 * (Tensor other)
 * (tuple of ints size, *, torch.device device)
 * (object data, *, torch.device device)

